.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -41,23 +91,51 @@ If you need to submit a pr for any of these repos during this period, you should
+ - [ ] **Freeze The Non Tagged Repositories** implement code freeze on the following repositories : 		
+	 - github-release-scripts https://github.com/adoptium/github-release-scripts
+	 - containers https://github.com/adoptium/containers
+	 - installer https://github.com/adoptium/installer
+	 - mirror-scripts https://github.com/adoptium/mirror-scripts
+
+- [ ] **Disable standard & evaluation nightly builds completely** by disabling the pipelines temporarily to free up resources and ensure no competing jobs during release week
+- [ ] **Disable standard & evaluation weekend builds completely** by disabling the pipelines temporarily to free up resources and ensure no competing jobs during the weekend prior to release, helps ensure no long running weekend jobs take critical resources during the beginning of the release period.
 - [ ] **Update aqaReference** to update with new git branch for 'aqaReference' to be used in release pipeline: https://github.com/adoptium/aqa-tests/branches (branch name to use should match the name of the [latest aqa-tests release](https://github.com/adoptium/aqa-tests/releases/latest))
+ - [ ] **Prepare For Release**
+>  - Double check the relevent aqa-tests branch version, ensure git branch for 'aqaReference' to be used in release pipeline: https://github.com/adoptium/aqa-tests/branches (branch name to use should match the name of the [latest aqa-tests release](https://github.com/adoptium/aqa-tests/releases/latest))
+>  - Update releaseVersions with release versions.
+>  - Update https://github.com/adoptium/mirror-scripts/blob/master/releasePlan.cfg
+> with expected tags, for more detail see
+> https://github.com/zdtsw/mirror-scripts/tree/issue/3167#skara-repos-and-processes
 - [ ] Generate release pipeline jobs for the new build scripts repo release tag, helper repo release tag and aqaReference: https://ci.adoptopenjdk.net/job/build-scripts/job/utils/job/release-build-pipeline-generator/build?delay=0sec
+
+**Wait For All Of The Above To Complete Successfully Before Proceeding!**
+
+- [ ] Log a helpdesk ticket with EF , to get all test materials updated
 - [ ] TC: Run the ProcessCheckMultiNode process cleaning job on all ci.role.test nodes, to ensure healthy state, verify all nodes successful: https://ci.eclipse.org/temurin-compliance/job/ProcessCheckMultiNode/build?delay=0sec
 - [ ] TC: Run the Setup_JCK_Run_Multinode job with CLEAN_DIR=true (to purge any old release contents/results) on all ci.role.test nodes, this will extract the jck_run folder with all the temurin.jtx exclude files, verify all nodes successful : https://ci.eclipse.org/temurin-compliance/job/Setup_JCK_Run_Multinode/build?delay=0sec
```

Comment:
I think the cleaning job needs running at the 2week point, as we are likely to hit "locked" files, as Shelley mentioned...